packages/loader/container-loader/src/container.ts

```diff
@@ -1149,8 +1160,9 @@ export class Container
 				this.mc.logger.sendTelemetryEvent(
 					{
 						eventName: "ContainerDispose",
-						// Only log error if container isn't closed
-						category: !this.closed && error !== undefined ? "error" : "generic",
+						// Use error category if there's an error, unless we already closed with an error (i.e. _criticalError is set)
+						category:
+							error !== undefined && this._criticalError === undefined ? "error" : "generic",
 					},
 					error,
```

Comment:
should we also log the close error, which could be different than the dispose error?

Reply:
Hmmm we could, but you can also just look at the ContainerClose event.  Leaning towards not.  I don't think this (both APIs getting called with an error) will happen commonly, if ever.

Reply:
Ok after trying to query for different cases of close and dispose, and wishing I had this easily available in aggregate, I'm adding this after all.